services-discovery/src/main/java/io/scalecube/services/discovery/ClusterAddresses.java

```diff
@@ -0,0 +1,34 @@
+package io.scalecube.services.discovery;
+
+import io.scalecube.services.transport.api.Address;
+import java.util.Arrays;
+
+public class ClusterAddresses {
+
+  private ClusterAddresses() {
+    // Do not instantiate
+  }
+
+  /**
+   * Converts one address to another.
+   *
+   * @param address scalecube services address
+   * @return scalecube cluster address
+   */
+  public static io.scalecube.transport.Address toAddress(
+      io.scalecube.services.transport.api.Address address) {
+    return io.scalecube.transport.Address.create(address.host(), address.port());
```

Comment:
we better consolidate all address to seperate jar so it can be used accross the modules

Reply:
This ClusterAddress utility was created  for purpose  of  integration  Microservices  type  Address and  scalecube  Cluster  Address, nothing more

Reply:
its created already several times and feels more like copy paste then solution

Reply:
Point of this refactoring is not to deal with messy Address types. Im very well aware of two Address classes. In this PR focus was on something else.